chromium_src/chrome/test/base/testing_profile.cc

```diff
@@ -0,0 +1,14 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/profiles/profile_util.h"
+
+#define BRAVE_INIT                            \
```

Comment:
`brave::CreateParentProfileData(this)` is needed here because during `TestingProfile::Init` which is called from `TestingProfile::TestingProfile`, will try to get supervised user settings service.

We also need to skip https://cs.chromium.org/chromium/src/chrome/test/base/testing_profile.cc?q=TestingProfile&sq=package:chromium&g=0&l=416-419 since setting the store again will hit DCHECK at https://cs.chromium.org/chromium/src/chrome/browser/supervised_user/supervised_user_settings_service.cc?type=cs&q=SupervisedUserSettingsService::Init&sq=package:chromium&g=0&l=86.